### PR TITLE
minor cosmetic tweak: add logos to loading screen

### PR DESCRIPTION
**Pull request by nike4613**
Just changes the ugly white text to some pretty logos. It uses Crunchyroll's own hosted logo for that portion.

### CODE REVIEW

**Comment by nike4613** [comment]
I would probably keep it actually playing after the document finishes loading, just because nobody likes a site that auto plays before it's finished loading. Still, showing the thumbnail is a good idea.

File: src/app/bootstrap.tsx [hunk 1]
```diff
@@ -116,25 +117,39 @@ class Bootstrap {
...
-          <polygon fill="#EBEBEB" points="256,386.153 255.801,386.206 205.227,372.55 201.994,336.333 177.419,336.333 156.409,336.333 162.771,407.634 255.791,433.457 256,433.399"/>
-          <path fill="#FFFFFF" d="M108.382,0h23.077v22.8h21.11V0h23.078v69.044H152.57v-23.12h-21.11v23.12h-23.077V0z"/>
-          <path fill="#FFFFFF" d="M205.994,22.896h-20.316V0h63.72v22.896h-20.325v46.148h-23.078V22.896z"/>
-          <path fill="#FFFFFF" d="M259.511,0h24.063l14.802,24.26L313.163,0h24.072v69.044h-22.982V34.822l-15.877,24.549h-0.397l-15.888-24.549v34.222h-22.58V0z"/>
-          <path fill="#FFFFFF" d="M348.72,0h23.084v46.222h32.453v22.822H348.72V0z"/>
-          <polygon fill="#FFFFFF" points="255.843,268.217 255.843,313.627 311.761,313.627 306.49,372.521 255.843,386.191 255.843,433.435 348.937,407.634 349.62,399.962 360.291,280.411 361.399,268.217 349.162,268.217"/>
-          <polygon fill="#FFFFFF" points="255.843,176.305 255.843,204.509 255.843,221.605 255.843,221.716 365.385,221.716 365.385,221.716 365.531,221.716 366.442,211.509 368.511,188.488 369.597,176.305"/>
-        </svg>
+      <div class="html5-loading-box-wrapper" style={style}>
```
Comment (YePpHa) [inline src/app/bootstrap.tsx]: I would rather show the thumbnail or show the logo not both. In this case I would prefer the thumbnail.

**Comment by YePpHa** [comment]
@nike4613 I would actually disagree with you here. Starting playing as soon as possible would be better. It will actually not start playing before the whole page has been rendered (all styles and scripts in `<head>` has been loaded or executed).

Whenever someone goes to a video their intent is to watch it. They don't want to wait for the comments or images on the page to finish loading before they can start watching. If we can reduce the amount of time between them clicking on the video link to start watching we should do so.
The `interactive` state means that the whole DOM tree has been loaded. However, the resources are not. This doesn't mean that the page has been rendered. It simply means that we have access to the DOM tree. If we add a video (with an instant buffering) and begin playing it. It would actually not start playing before the initial page rendering. The page starts rendering after the styles and scripts in the `<head>` have been loaded or executed (see https://developers.google.com/web/fundamentals/performance/critical-rendering-path/render-blocking-css).

In conclusion we should initialize the player on `interactive`. The player will not start playing before the page has been rendered.

The gif below shows the page rendering and when the area where the player is turns black is when the `interactive` is caught by the extension and the styles are added.

_This feature shown in the gif above is [Chrome Performance](https://developers.google.com/web/tools/chrome-devtools/evaluate-performance/) which is very useful for breaking down the performance of the page loading and if something is blocking the main thread._
